tests/capture-tests.js

```diff
@@ -243,26 +265,46 @@ require(["mobifyjs/utils", "capture"], function(Utils, Capture) {
         $("#qunit-fixture").append($iframe);
     });
 
+    // Ensure that a meta viewport tag is added to the document if one is not present
     asyncTest("ios8_0ScrollFix", function() {
-        var html = document.createElement('html');
-        var head = document.createElement('head');
-        html.appendChild(head);
+        var doc = makeDocument();
+
+        Capture.ios8AndGreaterScrollFix(doc, function() {
+            var meta = doc.getElementsByTagName('meta')[0];
+
+            ok(true,
+                'meta tag is appended');
+            equal(meta.getAttribute('name'), 'viewport',
+                'meta name is viewport');
+            equal(meta.getAttribute('content'), 'width=device-width', 'content is not width=device-width');
 
-        Capture.ios8AndGreaterScrollFix(html, function() {
-            var meta = html.getElementsByTagName('meta')[0]
+            start();
+        });
+    });
+
+    // Ensure that source document meta viewport tag is restored properly
```

Comment:
Now that we don't insert it if it's not capturing, we don't need this anymore do we?